src/callbacks/callbacks.jl

```diff
@@ -524,3 +526,16 @@ function gc_func(integrator)
     )
     return nothing
 end
+
+
+function update_surface_temp!(integrator)
+    d = 10 # ocean mixed layer depth [m]
```

Comment:
```suggestion
    depth = 10 # ocean mixed layer depth [m]
```
(I think it's more clear)